spark/v3.5/spark/src/main/java/org/apache/iceberg/spark/SparkTableUtil.java

```diff
@@ -92,6 +98,8 @@
 import org.apache.spark.sql.catalyst.plans.logical.LogicalPlan;
 import org.apache.spark.sql.execution.datasources.v2.DataSourceV2Relation;
 import org.apache.spark.sql.util.CaseInsensitiveStringMap;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
```

Comment:
why not javax.annotation.Nullable

Reply:
Actually this is probably good to have ... since we don't have NotNull in javax

Reply:
It looks we use both `java.annotation.Nullable` and `org.jetbrains.annotations.Nullable`